lib/src/main/java/co/nimblehq/alpine/lib/mlkit/MrzInfo.kt

```diff
@@ -0,0 +1,30 @@
+package co.nimblehq.alpine.lib.mlkit
+
+data class MrzInfo(
+    val documentNumber: String,
+    val dateOfBirth: String,
+    val dateOfExpiry: String
+) {
+
+    companion object {
+        fun createFrom(
+            documentNumber: String?,
+            dateOfBirth: String?,
+            dateOfExpiry: String?
+        ): MrzInfo? {
+            return if (
+                documentNumber != null
+                && dateOfBirth != null
+                && dateOfExpiry != null
+            ) {
+                MrzInfo(
+                    documentNumber,
+                    dateOfBirth,
+                    dateOfExpiry,
```

Comment:
Optional: Use named arguments here 😄 
```suggestion
                    documentNumber = documentNumber,
                    dateOfBirth = dateOfBirth,
                    dateOfExpiry = dateOfExpiry,
```